scormplayer/scorm-impl/service/src/java/org/sakaiproject/scorm/service/impl/ScormEntityProducer.java

```diff
@@ -347,6 +353,49 @@ private void updateCollectionDisplayName(String resourceId, String title) {
         }
     }
 
+    private void copyGradebookSettings(String fromContext, String toContext, ContentPackage source, ContentPackage copy) {
+        if (gradingService == null) {
+            return;
+        }
```

Comment:
we tend not to add null checks for services as there would be a larger issue if the spring couldn't inject the bean

```suggestion
```